autovalet_husky/src/autovalet_husky/find_parking_spot.py

```diff
@@ -132,6 +149,7 @@ def generate_parking_goal(self, tf, pos, rot):
         waypoint published to movebase
         '''
         tag2waypoint = PoseStamped()
+        tag2waypoint.header.frame_id = 'parking_spot'
```

Comment:
should be `self.aruco_frame_name`?

Reply:
great catch. Subbu had changed that for quick testing I think. And we forgot to revert the change.
Fixed it.